cmd/juju/gui/gui.go

```diff
@@ -175,7 +175,9 @@ func (c *guiCommand) Run(ctx *cmd.Context) error {
 		controllerName,
 	)
 
-	signalCh := make(chan os.Signal)
+	// Buffer size of 1 avoids race condition if signal is sent between
```

Comment:
A drive-by fix to get the Juju repo "go vet" clean (at least `go vet -composites=false`).